Tests/CookiesTests.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System.Net.Http.Headers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+using Moq;
+using Cookies = UI.Pages.Cookies;
+
+
+
+namespace Tests;
+
+[Collection(nameof(SliceFixture))]
+public class CookiesTests : CleanSliceFixture
+{
+    public CookiesTests(SliceFixture fixture) : base(fixture)
+    {
+    }
+
+    [Theory]
+    [InlineData(true, true)]
+    [InlineData(null, true)]
+    [InlineData(true, null)]
+    [InlineData(null, null)]
+    public void Cookies_OnGet_Test(bool? consent, bool? preferencesSet)
+    {
+        var httpContext = new DefaultHttpContext();
+        var modelState = new ModelStateDictionary();
+        var actionContext = new ActionContext(httpContext, new RouteData(), new PageActionDescriptor(), modelState);
+        var pageContext = new PageContext(actionContext);
+        Cookies cookiePage = new()
+        {
+            PageContext = pageContext
+        };
+        var result = cookiePage.OnGet(consent, preferencesSet);
+        result.Should().BeOfType<PageResult>();
+    }
+
+    [Fact]
+    public async void Cookies_For_FindATuitionPartner_Created_OnGet()
+    {
+        var cookiePage = await Fixture.GetPage<Cookies>()
+              .Execute(page =>
+              {
+                  page.PageContext = GetPageContext();
+                  page.OnGet(true, true);
```

Comment:
You also need to check that passing consent = false sets the cookie value to false

Reply:
The setting of false and true via the consent variable is done other tests